lib/livebook_web/live/session_live/popout_window_live.ex

```diff
@@ -0,0 +1,141 @@
+defmodule LivebookWeb.SessionLive.PopoutWindowLive do
```

Comment:
The popped out window creates a new connection to the server. Wouldn't it be better to fetch the data from the main window? Any tips to accomplish that?

Reply:
This is totally fine, it's the same as two users collaborating and everything is built under this assumption anyway. Pretty sure syncing everything on the client wouldn't really be viable.